### DISCUSSION TITLE
How do i remove the top tab bar?

### DISCUSSION THREAD

**Discussion by thecriticizedgamer** [piroor/treestyletab#3113] [Q&A]
Now that i have tree style tabs i don't need the top tab bar anymore so how do i get rid of it?

**Comment by irvinm** [comment]
userChrome.css:  https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-horizontal-tabs-at-the-top-of-the-window-1349-1672-2147
